### ISSUE TITLE
False positives for indented parens when the indentInsideParens option is used

### ISSUE THREAD

**Issue by evilebottnawi** [stylelint/stylelint#1451]
> Describe the issue. Is it a bug or a feature request (new rule, new option, etc.)?

Incorrect indentation

> Which rule, if any, is this issue related to?

`indentation`

> What CSS is needed to reproduce this issue?

e.g.

```css
header {
    background:
        linear-gradient(
            to bottom,
            transparentize($gray-dark, 1) 0%,
            transparentize($gray-dark, 0.1) 100%
        );
}
```

> What stylelint configuration is needed to reproduce this issue?

e.g.

```json
{
  "rules": {
    "indentation": [
            4,
            {
                "indentInsideParens": "once"
            }
        ],
  }
}
```

> Which version of stylelint are you using?

`6.6.0`

> How are you running stylelint: CLI, PostCSS plugin, Node API?

PostCSS

> Does your issue relate to non-standard syntax (e.g. SCSS, nesting, etc.)?

SCSS

> What did you expect to happen?

No error

> What actually happened (e.g. what warnings or errors you are getting)?

e.g. "The following warnings were flagged:"

```shell
test.css
4:13	⚠  Expected indentation of 8 spaces (indentation) [stylelint]
5:13	⚠  Expected indentation of 8 spaces (indentation) [stylelint]
6:13	⚠  Expected indentation of 8 spaces (indentation) [stylelint]
7:9	⚠  Expected indentation of 4 spaces (indentation) [stylelint]
```

**Comment by jeddy3** [comment]
Was this fine prior to `6.6`?

**Comment by jeddy3** [comment]
The following does not flag any warnings:

```css
header {
    background: linear-gradient(
        to bottom,
        transparentize($gray-dark, 1) 0%,
        transparentize($gray-dark, 0.1) 100%
    );
}
```

So, I think this is triggered when the `indentInsideParens` option is used and the actual parens are indented too.

**Comment by evilebottnawi** [comment]
@jeddy3 yep in 6.5 and lower this issue reproduced

**Comment by jeddy3** [comment]
Phew, for a moment there I thought we had celebrated the release a bit prematurely. I've very much got my figures crossed that this release doesn't come with too many hiccups :)

So this style of indenting (i.e. indenting the value below the property):

```css
header {
    background:
        linear-gradient(
            to bottom,
            transparentize($gray-dark, 1) 0%,
            transparentize($gray-dark, 0.1) 100%
        );
}
```

combined with the `indentInsideParens` option mustn't be used very much in the wild, as we've not had this bug reported for the whole time `6.5` was out.

**Comment by ntwb** [comment]
Related: #1149, #1458
